nucypher/utilities/prometheus/metrics.py

```diff
@@ -36,9 +36,12 @@
 import json
 from typing import List
 
-from prometheus_client.core import Timestamp
-from prometheus_client.registry import CollectorRegistry, REGISTRY
-from prometheus_client.utils import floatToGoString
+try:
+    from prometheus_client.core import Timestamp
+    from prometheus_client.registry import CollectorRegistry, REGISTRY
+    from prometheus_client.utils import floatToGoString
+except ImportError:
+    raise DevelopmentInstallationRequired(importable_name='prometheus_client')
```

Comment:
@KPrasch this was modified in #2100 based on our discussion.

Reply:
:+1: - Divergence needs a rebase here

Reply:
:heavy_check_mark: